main.js

```diff
@@ -1 +1,80 @@
-console.log('Hello world')
\ No newline at end of file
+{console.log('Hello world')
+let time, seconds, fullSec, fullMin, fullHour, clicked, turned;
+const frame = 60;
+
+const DOMS = {
+    startBtn: "#btn-start",
+    hours: "#hours",
+    minutes: "#minutes",
+    seconds: "#seconds",
+}
+
+const formatNumber = (number, time) =>
+{
+    if (typeof(number) === 'number' && !NaN) {
+        if (number < 10) {
+            document.querySelector(time).textContent = `0${number}`;
+        } else {
+            document.querySelector(time).textContent = `${number}`;
+        }
+    } else {
+        document.querySelector(time).textContent = `00`;
+    }
+}
+
+const countSec = () => {
+    fullSec = Math.floor(seconds/frame)
+    seconds++;
+    if (fullSec === 60){
+        seconds = 0;
+    }
+    formatNumber(fullSec, DOMS.seconds);
+}
+
+const countMin = () => {
+    fullMin++;
+    turned = false;
+    if (fullMin === 60) {
+        fullMin = 0;
+        turned = true;
+    }
+    formatNumber(fullMin, DOMS.minutes);
+}
+const countHour = () => {
+    fullHour++;
+    if (fullHour === 24) {
+        fullHour = 0;
+    }

```

Comment:
I love that you thought of this, but let's let the hours keep on ticking forever!